### DISCUSSION TITLE
Webpack ModuleFederationPlugin

### DISCUSSION THREAD

**Discussion by ILUSIODigital** [quasarframework/quasar#11524] [CLI - BEX mode]
Hello everyone,

I'm trying to set up Webpack ModuleFederationPlugin, which suppose to allow dynamically load component / store / or whatever. 
I did have a look on different configuration such as https://github.com/manfredsteyer/module-federation-plugin-example but so far didn't succeed to make it work. 

My first question before sharing more on my own webpack setup within extendWebpack or chainWebpack (I tried both) in quasar.conf.js will be :  is quasar initial webpack configuration allow the use of this plugin ?

**Comment by hawkeye64** [comment]
No idea. Never tried it or heard of someone else trying it, even with Vue.

**Comment by mingfang** [comment]
I'm looking into the same.
This is how it's done with plain Vite https://github.com/originjs/vite-plugin-federation, so people ARE doing it.
I'm trying to do the same with Quasar and webpack

**Reply by schupp13** [reply]
Have any of you got the module Federation to work in a quasar? I've been trying for a while now with no luck.
